docs/guide/tools/geometry.md

```diff
@@ -0,0 +1,78 @@
+# Geometry tools
+
+In the latest development version, there is a setsquare tool. A compass wheel is in work as well.
+
+## Setsquare
+
+The setsquare can be used for measuring lengths and angles and for geometric constructions.
+
+![Setsquare-Overview](../../img/guide/setsquare_overview.png)
+
+The longest side of the setsquare is used for measuring lengths and to draw lines aligned to it.
+Units are centimeters for lengths and degrees for angles. There are marks (in green) parallel 
```

Comment:
This sentence is a little awkward to me, how about the following?
```suggestion
Lengths are measured in centimeters, and angles are measured in degrees. There are marks (in green) parallel 
```

Reply:
I have incorporated that suggestion.